pkg/v1/mutate/mutate_test.go

```diff
@@ -266,6 +266,31 @@ func TestMutateConfig(t *testing.T) {
 	}
 }
 
+func TestMutateManifestAnnotations(t *testing.T) {
+	source := sourceImage(t)
+
+	newAnnotations := map[string]string{
+		"im.the.first.annotation": "hello world",
+	}
+
+	result, err := mutate.ManifestAnnotations(source, newAnnotations)
+	if err != nil {
+		t.Fatalf("failed to mutate a config: %v", err)
+	}
+
+	if manifestsAreEqual(t, source, result) {
```

Comment:
Does this indicate that the source was mutated in the process? I would expect source to be unmodified and the modified result to be returned.